Integrations/src/main/java/io/deephaven/integrations/python/PythonDeephavenSession.java

```diff
@@ -120,9 +124,20 @@ public PythonDeephavenSession(
         }
         scriptFinder = null;
 
+        registerJavaExecutor();
         publishInitial();
     }
 
+    private void registerJavaExecutor() {
+        // TODO (deephaven-core#4040) Temporary exec service until we have cleaner startup wiring
+        try (PyModule pyModule = PyModule.importModule("deephaven.server.executors");
+                final PythonDeephavenThreadsModule module = pyModule.createProxy(PythonDeephavenThreadsModule.class)) {
+            ExecutorService executorService = Executors.newFixedThreadPool(1);
```

Comment:
Don't we want all of our fancy python debugging support for this thread?

Reply:
Rebased (to get the changes where that was passed to this type) and added the thread initializer wiring